source/braille.py

```diff
@@ -2870,7 +2873,7 @@ def terminate(self):
 		super().terminate()
 		# Clear the display.
 		try:
-			self.display([0] * self.numCells)
+			handler.buffer.clear()

```

Comment:
This might need to be come the following, if the braille display is not cleared when terminating

```suggestion
			handler.mainBuffer.clear()
			handler._dismissMessage(shouldUpdate=True)
```